src-opam/linux.ml

```diff
@@ -121,10 +131,15 @@ module Apk = struct
 
   let dev_packages ?extra () =
     install
-      "build-base patch tar ca-certificates git rsync curl sudo bash \
-       libx11-dev nano coreutils xz ncurses-dev%s"
+      "build-base patch tar ca-certificates git rsync curl sudo bash nano \
+       coreutils xz ncurses-dev libX11-dev%s"
```

Comment:
On Alpine the name of packages are case sensitive
```suggestion
       coreutils xz ncurses-dev libx11-dev%s"
```
I would also suggest to keep the same ording as before to limit the diff and avoid such issues elsewhere